guides/source/active_record_multiple_databases.md

```diff
@@ -133,40 +133,40 @@ connection specification name.
 
 ```bash
-$ rails -T
-rails db:create                          # Creates the database from DATABASE_URL or config/database.yml for the ...
-rails db:create:animals                  # Create animals database for current environment
-rails db:create:primary                  # Create primary database for current environment
-rails db:drop                            # Drops the database from DATABASE_URL or config/database.yml for the cu...
-rails db:drop:animals                    # Drop animals database for current environment
-rails db:drop:primary                    # Drop primary database for current environment
-rails db:migrate                         # Migrate the database (options: VERSION=x, VERBOSE=false, SCOPE=blog)
-rails db:migrate:animals                 # Migrate animals database for current environment
-rails db:migrate:primary                 # Migrate primary database for current environment
-rails db:migrate:status                  # Display status of migrations
-rails db:migrate:status:animals          # Display status of migrations for animals database
-rails db:migrate:status:primary          # Display status of migrations for primary database
-rails db:schema:dump                     # Creates a db/schema.rb file that is portable against any DB supported  ...
-rails db:schema:dump:animals             # Creates a db/schema.rb file that is portable against any DB supported  ...
-rails db:schema:dump:primary             # Creates a db/schema.rb file that is portable against any DB supported  ...
-rails db:schema:load                     # Loads a schema.rb file into the database
-rails db:schema:load:animals             # Loads a schema.rb file into the animals database
-rails db:schema:load:primary             # Loads a schema.rb file into the primary database
-rails db:structure:dump                  # Dumps the database structure to db/structure.sql. Specify another file ...
-rails db:structure:dump:animals          # Dumps the animals database structure to sdb/structure.sql. Specify another ...
-rails db:structure:dump:primary          # Dumps the primary database structure to db/structure.sql. Specify another ...
-rails db:structure:load                  # Recreates the databases from the structure.sql file
-rails db:structure:load:animals          # Recreates the animals database from the structure.sql file
-rails db:structure:load:primary          # Recreates the primary database from the structure.sql file
+$ bin/rails -T
+bin/rails db:create                          # Creates the database from DATABASE_URL or config/database.yml for the ...
+bin/rails db:create:animals                  # Create animals database for current environment
+bin/rails db:create:primary                  # Create primary database for current environment
+bin/rails db:drop                            # Drops the database from DATABASE_URL or config/database.yml for the cu...
```

Comment:
Are these actual output for `bin/rails -T`?

Reply:
@kamipo whoops, no that is not the actual output from that command.

I had a look and could not find a way to prefix `bin/` to that command, I think it calls out to rake here `railties/lib/rails/commands/rake/rake_command.rb` but not 100% sure.